### DISCUSSION TITLE
can't clear a dmca'ed notification {help needed}

### DISCUSSION THREAD

**Discussion by bachig26** [community/community#30293] [Other Feature Feedback, Questions, & Ideas]
One of a Repo that I have subscribed for notification got DMCA'ed and I might had a notification from it, now I can't clear them from my notifications.

It's still shows as notification in the icon but inside it's a hidden or ghost notification at this point.

I want to know how to clear/solve/reset this.

#### Screenshot
![cs3](https://user-images.githubusercontent.com/96393348/185795701-ba676679-a17e-428c-815e-dd4d9197c06b.JPG)

**Comment by airtower-luna** [comment]
That looks really annoying, I'd send a [bug report to GitHub support](https://support.github.com/contact/bug-report). I hope you get it fixed soon!

**Reply by bachig26** [reply]
> That looks really annoying, I'd send a [bug report to GitHub support](https://support.github.com/contact/bug-report). I hope you get it fixed soon!

thank you... hope to see this resolved.

**Comment by jfullstackdev** [comment]
that's a common one, it's called ghost notification, 

this helped me twice, it also the answer from the Support

https://github.com/community/community/discussions/6874#discussioncomment-2859125

**Reply by bachig26** [reply]
thanks, i'll try it.

**Reply by bachig26** [reply]
curl -X PUT
    -H "Accept: application/vnd.github.v3+json" \
    -H "Authorization: token $TOKEN" \
     https://api.github.com/notifications \
    -d '{"last_read_at":"2022-05-31T00:00:00Z"}'

so, for this last read time, just the current time or for hour, minute, second should at exact current one when press enter? i tried once but didn't work.
can you help me?

**Reply by airtower-luna** [reply]
It doesn't need to be the exact current time, just relative to which time you want to mark all older notifications as read. Mind that the date in your example is pretty old. :wink: 

And don't forget to set `"read": true` as well!

**Reply by bachig26** [reply]
"read": true?

**Reply by airtower-luna** [reply]
The second parameter you need to send in the payload, in addition to `last_read_at`. See the documentation: https://docs.github.com/en/rest/activity/notifications#mark-notifications-as-read

To mark everything before 00:00 UTC today as read, the payload would need to be something like:

```json
{"last_read_at": "2022-10-05T00:00:00Z", "read": true}
```

**Reply by bachig26** [reply]
sorry to distrub again and again, I tried using this command with my token, but I'm getting this error that i provided below.

curl -X PUT     -H "Accept: application/vnd.github.v3+json" \     -H "Authorization: token $TOKEN" \      https://api.github.com/notifications \     -d '{"last_read_at": "2022-10-05T00:00:00Z", "read": true}'


#### Error Message:

`Invoke-WebRequest : Cannot bind parameter 'Headers'. Cannot convert the "Accept:`
`application/vnd.github.v3+json" value of type "System.String" to type`
`"System.Collections.IDictionary".`
`At line:1 char:20`
`+ curl -X PUT     -H "Accept: application/vnd.github.v3+json" \     -H  ...`
`+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~`
`    + CategoryInfo          : InvalidArgument: (:) [Invoke-WebRequest], ParameterBindingException`
`    + FullyQualifiedErrorId : CannotConvertArgumentNoMessage,Microsoft.PowerShell.Commands.InvokeWeb
   RequestCommand`

**Reply by airtower-luna** [reply]
That looks like a Windows/Powershell problem. If you use Git you probably have Git Bash on your system, try to run the command there. AFAIK the `$TOKEN` syntax would not work their either. :wink:

**Reply by jfullstackdev** [reply]
if you have Codespace, you can use that, no further setup is needed
